Sources/Profile-Modify.php

```diff
@@ -1199,7 +1199,15 @@ function makeCustomFieldChanges($memID, $area, $sanitize = true, $returnErrors =
 			// Any masks?
 			if ($row['field_type'] == 'text' && !empty($row['mask']) && $row['mask'] != 'none')
 			{
-				if ($row['mask'] == 'email' && (!filter_var($value, FILTER_VALIDATE_EMAIL) || strlen($value) > 255))
+				if($row['mask'] == 'nohtml' && ($value != strip_tags($value) || (!filter_var($value, FILTER_SANITIZE_STRING))))
```

Comment:
nit: `if(` -> `if (`

Reply:
Should be comparing return value of filter_var to $value as the return value of filter_var can be a False-y value.